main.nf

```diff
@@ -283,6 +284,69 @@ process intersect_files {
     """
 }
 
+
+vep_infiles = masked_beds_vep.mix(vcfs)
+
+// TODO: Figure out running characteristics
+process variant_effect_predictor {
+    input:
+        file infile from vep_infiles
+    output:
+        file '*.vep'
+
+    publishDir params.outdir, mode: 'copy'
+
+    // We only need one core for this part
+    if ( nextflow_running_as_slurmjob() ) {
+        executor 'local'
+    }
+    else {
+        executor 'slurm'
+        queue 'core'
+        time params.short_job
+    }
+
+    module 'bioinfo-tools'
+    module "$params.modules.vep"
+
+    script:
+    """
+    case "$infile" in
+        *vcf) FORMAT=vcf ;;
+        *bed) FORMAT=ensembl ;;
+        *)    printf "Unrecognized format for '%s'" "$infile" >&2
+              exit 1;;
+    esac
+
+    OUTFILE=\$( echo $infile | sed 's/\$/.vep/' )
+    ## TODO: Figure out if cache is needed
+    VEP_CACHE=/sw/data/uppnex/vep/84
+    variant_effect_predictor.pl \
+        -i $infile               \
+        --format \$FORMAT        \
+        -cache --dir \$VEP_CACHE \
+        -o \$OUTFILE             \
+        --vcf                    \
```

Comment:
If `$FORMAT`, `$VEP_CACHE` and `$OUTFILE` are evaluated by the shell (and not by Groovy), then quote them.